### PR TITLE
EPMRPP-40810 Filter entities description validation fix. Fix filters/members search

### PR DESCRIPTION
**Pull request by AmsterGet**

### CODE REVIEW

File: app/src/pages/inside/filtersPage/filterPageToolbar/filterPageToolbar.jsx [hunk 1]
```diff
@@ -30,10 +30,8 @@ const messages = defineMessages({
   validate: ({ filter }) => ({
     filter: filter && filter.length < 3 ? 'filterNameError' : undefined,
   }),
-  onChange: (values, dispatch, props, previousValues) => {
-    if (typeof previousValues.filter === 'undefined') {
-      return;
-    }
+  enableReinitialize: true,
+  onChange: (values, dispatch, props) => {
     if (!values.filter || values.filter.length >= 3) {
```
Comment (ElenaRomanchuk) [inline app/src/pages/inside/filtersPage/filterPageToolbar/filterPageToolbar.jsx]: Let's use common validator for this and [that row](https://github.com/reportportal/service-ui/pull/1964/files#diff-e984a29f971b56d9cb7558ab94889c44L31)

File: app/src/pages/inside/filtersPage/filterPageToolbar/filterPageToolbar.jsx [hunk 2]
```diff
@@ -44,43 +42,41 @@ const messages = defineMessages({
 export class FilterPageToolbar extends React.Component {
   static propTypes = {
     intl: intlShape,
-    change: PropTypes.func,
     invalid: PropTypes.bool,
-    filter: PropTypes.string,
-    filters: PropTypes.array,
+    disabled: PropTypes.bool,
```
[the remark this answers is not in the document]
Comment (AmsterGet) [reply app/src/pages/inside/filtersPage/filterPageToolbar/filterPageToolbar.jsx]: Fixed

File: app/src/pages/inside/filtersPage/filterPageToolbar/filterPageToolbar.jsx (hunk 1 above)
Comment (AmsterGet) [reply app/src/pages/inside/filtersPage/filterPageToolbar/filterPageToolbar.jsx]: Fixed

File: app/src/pages/inside/filtersPage/filterPageToolbar/filterPageToolbar.jsx [hunk 3]
```diff
@@ -44,43 +42,41 @@ const messages = defineMessages({
...
     onAddFilter: PropTypes.func,
   };
 
   static defaultProps = {
     intl: {},
     invalid: false,
-    filter: null,
-    filters: [],
-    change: () => {},
+    disabled: null,
```
[the remark this answers is not in the document]
Comment (AmsterGet) [reply app/src/pages/inside/filtersPage/filterPageToolbar/filterPageToolbar.jsx]: Fixed

File: app/src/common/utils/validation.js [hunk 4]
```diff
@@ -17,7 +17,7 @@ export const widgetName = (value) => value && value.length >= 3 && value.length
 export const itemNameEntity = (value) => value.length >= 3 && value.length <= 256;
 export const launchNumericEntity = (value) =>
   value.length >= 1 && value.length <= 18 && !!value.match(/^[0-9]+$/);
-export const launchDescriptionEntity = (value) => value.length >= 3 && value.length <= 18;
+export const descriptionEntity = (value) => value.length >= 3 && value.length <= 18;
```
[the remark this answers is not in the document]
Comment (AmsterGet) [reply app/src/common/utils/validation.js]: Fixed

File: app/src/pages/inside/filtersPage/filterPageToolbar/filterPageToolbar.jsx [hunk 5]
```diff
@@ -13,6 +13,8 @@ import styles from './filterPageToolbar.scss';
 
 const cx = classNames.bind(styles);
 
+const searchFilterValidator = (filter) => !filter || filter.length >= 3;
```
Comment (ElenaRomanchuk) [inline app/src/pages/inside/filtersPage/filterPageToolbar/filterPageToolbar.jsx]: May be isSearchFilterValid would provide more information about it's logic?

Comment (AmsterGet) [reply app/src/pages/inside/filtersPage/filterPageToolbar/filterPageToolbar.jsx]: @ElenaRomanchuk 
`isSearchFilterValid` sounds like a boolean variable, not a function.
What you think about `validateSearchFilter` ?

Comment (ElenaRomanchuk) [reply app/src/pages/inside/filtersPage/filterPageToolbar/filterPageToolbar.jsx]: I think it's OK

**Pull request merged**